Source/Xamarin/Prism.Forms/PrismApplicationBase.cs

```diff
@@ -136,6 +138,11 @@ protected virtual void InitializeModules()
         protected abstract void ConfigureContainer();
 
         /// <summary>
+        /// Finish the container's configuration after all other types are registered.
+        /// </summary>
+        protected virtual void FinishContainerConfiguration() {}
```

Comment:
Might be needed for other containers we're not supporting but people want to implement as well.

What alternative do you suggest? Highjacking the InitializeModules override in the Autofac implementation? Isn't very clean.